Tribes/src/app/statusboard/statusboard.component.html

```diff
@@ -1,4 +1,44 @@
 <h1>STATUSBOARD</h1>
 <div>
     <app-resources></app-resources>
+</div>
+
+<div class="container-fluid">
+    <div>
+        <ul>
+            <li *ngFor="let building of listOfBuildings">
+                <div *ngIf="building.buldingTypeENUM ==='MINE'">
```

Comment:
ugyanugy itt is, szerintem az Enumos verzio mar nem OK

Reply:
done